k8s/bases/swe-d2cc/kustomization.yaml

```diff
@@ -23,4 +23,8 @@ resources:
   - swe-d2cc-rao-logs-dispatcher.yaml
   - swe-d2cc-runner.yaml
   - swe-d2cc-runner-itools-configmap.yaml
-  - swe-d2cc-task-manager.yaml
\ No newline at end of file
+  - swe-d2cc-task-manager.yaml
+  - swe-d2cc-boundary-minio-automatic-purge.yaml
+  - swe-d2cc-cgm-minio-automatic-purge.yaml
+  - swe-d2cc-crac-minio-automatic-purge.yaml
+  - swe-d2cc-glsk-minio-automatic-purge.yaml
```

Comment:
I think we should also clean the artifacts and the outputs directory that contains a lot of file. They are stored in different folders Years/month/day.

Reply:
:+1: